### PR TITLE
Add support for Hotspot GC options

### PR DESCRIPTION
**Pull request by AlenBadel**
Adding documentation to include the following new options:
-XX:ParallelCMSThreads
-XX:ConcGCThreads
-XX:ParallelGCThreads

Signed-off-by: AlenBadel <Alen.Badel@ibm.com>

### CODE REVIEW

**Comment by AlenBadel** [comment]
See Related Issue: https://github.com/eclipse/openj9-docs/issues/453

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by AlenBadel** [comment]
Jenkins doc stage

**Comment by AlenBadel** [comment]
This is ready to review. I would appreciate if you could look through it. @SueChaplain

File: docs/cmdline_migration.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ You can use the following command-line options in OpenJ9, just as you did in Hot
 | [`-Xrs`](xrs.md)                                                 | Prevents the OpenJ9 run time environment from handling signals.                                                                              |
 | [`-Xss`](xss.md)                                                 | Sets the Java&trade; thread stack size. (Equivalent to `-XX:ThreadStackSize`). <i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** Unlike HotSpot, this option applies only to the Java stack. OpenJ9 has a separate native stack for operating system threads (see [`-Xmso`](xmso.md))  |
 | [`-Xverify:mode`](xverify.md)                                    | Enables or disables the verifier.                                                                                                            |
+| [`-XX:ConcGCThreads`](xxconcgcthreads.md)                        | Configure number of GC mutator background threads.                                                                                           |
```
Comment (SueChaplain) [inline docs/cmdline_migration.md]: Please say "Configures the number of ....." for parity with the rest of the table contents. Please also amend for the other 2 options added.

File: docs/version0.18.md [hunk 2]
```diff
@@ -99,6 +99,13 @@ Support for Concurrent scavenge mode is now extended to macOS. For more informat
 
 The previously undocumented option [`-Xgc:noConcurrentScavenge`](xgc.md#noconcurrentscavenge) disables pause-less garbage collection.
 
+### Support for OpenJDK HotSpot options
```
Comment (SueChaplain) [inline docs/version0.18.md]: Please add this to the list at the top of the topic as per other items in the release notes.

File: docs/xxconcgcthreads.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:ConcGCThreads
+
+This Oracle Hotspot option affects the number of threads the concurrent garbage collector will use. This option is recognized by OpenJ9 and provided for compatibility.
```
Comment (SueChaplain) [inline docs/xxconcgcthreads.md]: Please avoid the use of future tense. Perhaps re-phrase this sentence " This Oracle HotSpot option affects the number of threads that are used by the current garbage collector." Please also note the accurate spelling of HotSpot, with upper case "S".

File: docs/xxconcgcthreads.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## Syntax
+
+        -XX:ConcGCThreads=<number>
+
+Where `<number>` is the number of low-priority background threads attached to assist the mutator threads in concurrent mark.
```
Comment (SueChaplain) [inline docs/xxconcgcthreads.md]: "..... low-priority background threads that are attached to ....."

File: docs/xxparallelgcthreads.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:ParallelGCThread
+
+This Oracle Hotspot option specifies the number of threads used during parallel operations of the default garbage collector. This option is recognized by OpenJ9 and provided for compatibility.
```
Comment (SueChaplain) [inline docs/xxparallelgcthreads.md]: " .... specifies the number of threads that are used during .... "

File: docs/xxparallelcmsthreads.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:ParallelCMSThreads
+
+This Oracle Hotspot option affects the number of threads the concurrent garbage collector will use. This option is recognized by OpenJ9 and provided for compatibility.
```
Comment (SueChaplain) [inline docs/xxparallelcmsthreads.md]: Please avoid future tense and rework similar to last comment.

Comment (SueChaplain) [reply docs/xxparallelcmsthreads.md]: And HotSpot spelling too please.

File: docs/xxparallelgcthreads.md (hunk 5 above)
Comment (SueChaplain) [reply docs/xxparallelgcthreads.md]: Hotspot / HotSpot

File: docs/version0.18.md (hunk 2 above)
Comment (AlenBadel) [reply docs/version0.18.md]: My apologies, didn't see the topics indices. 
See change here:
https://github.com/eclipse/openj9-docs/pull/464/commits/27ad960b4d2a8bb5eb0a9435ed38c03d1c7fc5ca

File: docs/cmdline_migration.md (hunk 1 above)
Comment (AlenBadel) [reply docs/cmdline_migration.md]: See 
https://github.com/eclipse/openj9-docs/pull/464/commits/1bccc4d24e6b98d4966aa1e3492a812dda6eb391
https://github.com/eclipse/openj9-docs/pull/464/commits/0c09eaea9661f8789a01655e01df0c3d8173d6b3

File: docs/xxconcgcthreads.md (hunk 3 above)
Comment (AlenBadel) [reply docs/xxconcgcthreads.md]: Understood. Will consider this in the future. 
See https://github.com/eclipse/openj9-docs/pull/464/commits/eb7fb597353edbac68539e9012f04954857849b0

File: docs/xxconcgcthreads.md (hunk 4 above)
Comment (AlenBadel) [reply docs/xxconcgcthreads.md]: See https://github.com/eclipse/openj9-docs/pull/464/commits/eb7fb597353edbac68539e9012f04954857849b0

File: docs/xxparallelgcthreads.md (hunk 5 above)
Comment (AlenBadel) [reply docs/xxparallelgcthreads.md]: See https://github.com/eclipse/openj9-docs/pull/464/commits/54b94909a0a7bebb591926c55485485575771214

File: docs/xxparallelcmsthreads.md (hunk 6 above)
Comment (AlenBadel) [reply docs/xxparallelcmsthreads.md]: See https://github.com/eclipse/openj9-docs/pull/464/commits/094dc778d3f9c2c3f1e8f9758dfef579be8f1759

**Comment by AlenBadel** [comment]
@SueChaplain Thanks for the feedback. If you're satisfied with the changes and the comments have been resolved I'll squash all my commits.

**Comment by pshipton** [comment]
This is documentation for https://github.com/eclipse/openj9/pull/7723 which is included in the 0.18.0 release, so this should be merged for this release @SueChaplain

**Pull request merged**